swagger.yml

```diff
@@ -3066,6 +3077,83 @@ components:
           x-go-name: PerAPI
       type: object
       x-go-package: github.com/TykTechnologies/tyk/user
+    RateLimitSmoothing:
+      properties:
+        delay:
+          type: integer
+          format: int64
+          description: Delay is the minimum time between rate limit changes (in seconds).
+          x-go-name: Delay
+        enabled:
+          type: boolean
+          description: Enabled indicates if rate limit smoothing is active.
+          x-go-name: Enabled
+        step:
+          type: integer
+          format: int64
+          description: Step is the increment/decrement for adjusting the rate limit.
+          x-go-name: Step
+        threshold:
+          type: integer
+          format: int64
+          description: Threshold is the request rate above which smoothing is applied.
+          x-go-name: Threshold
+        trigger:
```

Comment:
we can specify boundary values for this right? (0,1)

Reply:
added minimums, steps, technically one can use >= 1, the math allows it